test-utils/test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/random/RandomGeneratorTest.java

```diff
@@ -276,8 +277,41 @@ public void shouldAlwaysPickSameElementFromAnEnumWithSingleElement() {
         final Generator<SingleEnum> enumGenerator = randomEnum(SingleEnum.class);
 
         // then
-        typeCheck(enumGenerator, s -> enumGenerator.next().compareTo(enumGenerator.next()) == 0).verify(times(NUMBER_OF_TIMES));
+        typeCheck(enumGenerator, s -> enumGenerator.next().compareTo(enumGenerator.next()) == 0)
+                        .verify(times(NUMBER_OF_TIMES));
     }
 
-    enum SingleEnum {SINGLE_VALUE}
-}
\ No newline at end of file
+    @Test
+    public void shouldReturnAnInstanceEmailAddressGenerator() {
```

Comment:
True, they don't add much value like the other tests, after the refactor moved the functionality to each generator. 
They are superficial tests to test the reference exists in the RandomGenerator and has not been removed in a refactor.